automatic/waterfox/Readme.classic.md

```diff
@@ -0,0 +1,27 @@
+# <img src="https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-coreteampackages@2171e76d9468526e4e792d20ac6b68e1a8fdc93a/icons/waterfox.png" width="48" height="48"/> [waterfox](https://chocolatey.org/packages/waterfox)
+
+## How and why Waterfox is here...Waterfox Classic

```

Comment:
Do we need a section header for this?
Maybe just making the paragraph bold would be better instead?

Not completely sure, it just feels off to me.